src/app.js

```diff
@@ -1 +1,27 @@
 'use strict';
+
+import fs from 'fs'
+import { createInterface } from 'readline';
+
+const rl = createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const createFile = () => {
+  rl.question('file name: ',(fileName) => {
+    rl.question('file content: ',(fileContent) => {
+      fs.writeFile(`./${fileName}.txt`, fileContent, (err) => {
+        if(err) {
+          console.log(`Error: ${err}`)
+        } else {
+          console.log(`file created with a name of '${fileName}'`)
+        }
+        rl.close();
+      });
+    });
+  });
+}
```

Comment:
any ideas on how you can get rid of nesting like that? It is reeeeally difficult to read

Reply:
or at least make it more readable. I mean, you can create some helpers and use them here